CHANGELOG.md

```diff
@@ -17,6 +17,8 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 - [Multiple Datasource] Add TLS configuration for multiple data sources ([#6171](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6171))
 
+- [Workspace] Add delete saved objects by workspace functionality([#6013](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6013))
+
```

Comment:
```suggestion
```
Seems CHANGELOG.md should not have change within backport PR. Release owner will help to collect all the changes inside that release and batch update CHANLOG. 

Better to remove this CHANGE.